discord/client.py

```diff
@@ -2043,17 +2043,75 @@ async def fetch_skus(self) -> list[SKU]:
         data = await self._connection.http.list_skus(self.application_id)
         return [SKU(data=s) for s in data]
 
-    async def fetch_entitlements(self) -> list[Entitlement]:
+    async def fetch_entitlements(
+        self,
+        user: Snowflake | None = None,
+        skus: list[Snowflake] | None = None,
+        before: SnowflakeTime | None = None,
+        after: SnowflakeTime | None = None,
+        limit: int | None = 100,
+        guild: Snowflake | None = None,
+        exclude_ended: bool = False,
+    ) -> EntitlementIterator:
         """|coro|
 
         Fetches the bot's entitlements.
 
         .. versionadded:: 2.5
 
+        .. versionchanged:: 2.6
+
+            Added new parameters
+
+        Parameters
+        ----------
+        user: :class:`.abc.Snowflake` | None
```

Comment:
Rest of the library uses `Optional[T]` in docstrings, should either stick to that or use this notation all over.

Reply:
I'm aware it's been used, however I'm unaware of any decision to migrate. Both notations are just displayed as text in the documentation anyway, therefore I'm fine with either as long as we stay consistent.